cmd/checkmetrics/history/history.sh

```diff
@@ -0,0 +1,182 @@
+#!/usr/bin/env bash
+#
+# Copyright (c) 2019 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+
+# Extract Jenkins metrics server historic data.
+# Useful when re-setting the checkmetrics baseline data.
+
+set -e
+
+basedir=$(dirname "$0")
+
+# How many recent builds do we evaluate
+NUM_BUILDS=5
+
+# What is the default set of repos (Jenkins jobs) we evaluate
+default_repos=()
+default_repos+=("kata-metrics-runtime-ubuntu-16-04-PR")
+default_repos+=("kata-metrics-tests-ubuntu-16-04-PR")
+repos=()
+
+# What test results do we evaluate for each build
+tests=()
+test_queries=()
+tests+=("boot-times")
+test_queries+=(".\"boot-times\".Results | [.[] | .\"to-workload\".Result] | add / length")
+
+tests+=("memory-footprint")
+test_queries+=(".\"memory-footprint\".Results | .[] | .average.Result")
+
+tests+=("memory-footprint-ksm")
+test_queries+=(".\"memory-footprint-ksm\".Results | .[] | .average.Result")
+
+resultsdir="${basedir}/results"
```

Comment:
Could you mention this in the usage and maybe provide an option / variable to allow it to be changed?